### PR TITLE
Changed from Atom to VSCode

### PR DESCRIPTION
**Pull request by ijemmao**
* Changed GIF and atom references

### CODE REVIEW

**Review by timofei7 — CHANGES_REQUESTED** [review]
it'd be nice to show how to merge them in VScode, a gif or a screencap.  I know vscode has it builtin but reminding them would be nice

File: main.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /* SETUP */
-var map = L.map('map', {
+let map = L.map('map', {
```
Comment (timofei7) [inline main.js]: we should leave this as `var` as it won't work in some standard browsers that don't support es6

Comment (timofei7) [reply main.js]: probably not many there but still, lets just keep it in vanilla js

File: README.md [hunk 2]
```diff
@@ -140,9 +140,6 @@ var h = 'Hi!';
 
 where the line between  `<<<HEAD` and `====` is the line/lines of code that are relevant in your file, and between  `====` and `>>>> cb1abc6` are from the remote repository (the repository that you see on github.com). The crazy characters refer to the commit id hash that the change comes from.
```
Comment (timofei7) [inline README.md:144]: a demonstration image would be nice

**Review by timofei7 — CHANGES_REQUESTED** [review]

**Comment by ijemmao** [comment]
@timofei7 Oh my bad, I forgot to include the gif. The gif should be in the file in the latest commit.

**Review by timofei7 — APPROVED** [review]

**Pull request merged**